README.md

```diff
@@ -0,0 +1,82 @@
+# Sign In With Ethereum
+
+This crate provides a pure Rust implementation of EIP-4631: Sign In With Ethereum.
+
+## Installation
+
+SIWE can be easily installed in any Rust project by including it in said project's `cargo.toml` file:
+
+``` toml
+siwe = "0.1"
+```
+
+## Usage
+
+SIWE exposes a `Message` struct which implements EIP-4361.
+
+### Parsing a SIWE Message
+
+Parsing is done via the `Message` implementation of `FromStr`:
+
+``` rust
+let message: Message = string_message.parse()?;
+```
+
+### Verifying and Authenticating a SIWE Message
+
+Verification and Authentication is performed via EIP-191, using the `.address` field of the `Message` as the expected signer:
```

Comment:
Minor point, but the field is `address` (without the leading period) in the EIP, and I think it doesn't make a difference from a rust POV to simply call it `address`.

Reply:
done